tests/tasks/get_profile_stat.yml

```diff
@@ -24,6 +26,30 @@
   ignore_errors: yes
   changed_when: false
 
-- name: Set NM profile exist flag based on the nmcli output
-  set_fact: lsr_net_profile_exists=true
+# lsr_net_profile_ansible_managed:
+# under NetworkManager's control, the comment is not added by design.
+# Thus, set it always to true.
+- name: >-
+    Set NM profile exist and true ansible_managed comment flag
+    based on the nmcli output
```

Comment:
There is also a possibility to add some kind of comments to keyfiles via the NM API. If it is considered to be useful, then the role could be updated to do this, too.

Reply:
> There is also a possibility to add some kind of comments to keyfiles via the NM API. If it is considered to be useful, then the role could be updated to do this, too.

Maybe, but the intention is to have the `ansible_managed` header in files that are directly managed by the role.  Managing the keyfiles via the NM API sounds like some level of indirection, so I'm not sure.

Reply:
So, is it ok to leave the ifcfg file generated by NetworkManager without the "this file is managed by NM" type of comment? I searched around the ifcfg-rh plugin, but I could not find a way to add it...

Reply:
If there is no way to do it, then I think we should not do it

Reply:
ok